README.md

```diff
@@ -130,7 +130,22 @@ For more information on defining your transformation, please consult [ImageMagic
 
 ## Storage of files
 
-Arc currently only supports Amazon S3 as a destination for file uploads.
+Arc currently supports Amazon S3 and local destinations for file uploads.
+
+### Local Configuration
+```elixir
+defmodule Avatar do
+  use Arc.Definition
+
+  @versions [:original, :thumb]
```

Comment:
Can you remove the `@versions` declaration, or add a custom filename to this example module?  Same as your test case - specifying multiple versions without a custom filename function will result in the versions overriding each other.